dist/origin-web-catalogs.js

```diff
@@ -293,25 +293,50 @@ webpackJsonp([ 0, 1 ], [ function(e, t) {
             id: "middleware",
             label: "Middleware",
             subCategories: [ {
-                id: "jboss",
-                label: "JBoss",
-                tags: [ "jboss" ],
-                icon: "font-icon icon-openjdk"
+                id: "amq",
+                label: "A-MQ",
+                tags: [ "amq" ],
+                icon: "font-icon icon-jboss"
+            }, {
+                id: "processserver",
+                label: "BPM Suite",
+                tags: [ "processserver" ],
+                icon: "font-icon icon-jboss"
+            }, {
+                id: "decisionserver",
+                label: "BRMS",
+                tags: [ "decisionserver" ],
+                icon: "font-icon icon-jboss"
+            }, {
+                id: "datagrid",
+                label: "Data Grid",
+                tags: [ "datagrid" ],
+                icon: "font-icon icon-jboss"
+            }, {
+                id: "datavirt",
+                label: "Data Virtualization",
```

Comment:
@knrc Is there a shorter label that we could use here? "Data Virt" maybe? This makes the subcategory tab larger than others.